Makefile

```diff
@@ -183,10 +183,12 @@ KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 GINKGO ?= $(LOCALBIN)/ginkgo
+OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
 CONTROLLER_TOOLS_VERSION ?= v0.11.1
+OPERATOR_SDK_VERSION ?= v1.26.1
```

Comment:
other operators are using v1.31.0:
* https://github.com/openstack-k8s-operators/nova-operator/blob/main/Makefile#L207
* https://github.com/openstack-k8s-operators/install_yamls/blob/9db37fca2ce7c2a866df363ee14ef97aff731127/devsetup/roles/download_tools/defaults/main.yaml#L9

Reply:
updated, I updated this to point to v1.31.0